api_reference/source/utils.py

```diff
@@ -0,0 +1,58 @@
+# Define the modules and members to include in Sphinx documentation
+INCLUDE_MEMBERS = {
```

Comment:
Skip_member is not yet being used and unsure if a script is completely necessary to filter parts of the documentation automatically. It could lead to maintenance workload down the road so it might be easier to just select the parts of the documentation manually? Can set up file structure to make manual selection easier.